### ISSUE TITLE
Add djinni distribution in binary form

### ISSUE THREAD

**Issue by Mr-Goldberg** [dropbox/djinni#446]
Please add djinni binaries for windows/linux/mac so developer can just "download and use".
It is not possible to build djinni on some machines. I think this is location-related. Internet accessibility is fine, but djinni downloads during build may take more than 24hrs and still be in progress, moving veeery sloooow.
Screenshot shows that the downloads are progressing, but super-slow.
![picturemessage_g20vmh34 qmk](https://user-images.githubusercontent.com/23424480/60882019-a96b2c00-a24f-11e9-8073-43d9205d550d.png)

**Comment by a1lu** [comment]
You can build it by your own on a machine where it's working:
 https://github.com/dropbox/djinni#generate-a-standalone-jar

**Comment by Mr-Goldberg** [comment]
Thanks, I will definitely do it. But it is still great to have an option to just download it and use.

**Comment by a1lu** [comment]
That's true, but unfortunately the repository is not maintained any more.

> Maintenance note: This repo is stable but no longer actively maintained. No further development is planned, and no new feature PRs will be merged. Bug fixes may be merged on a volunteer basis. If you have questions or want to talk to other users of Djinni, you can join the Slack community via the link at the end of this document.

**Comment by artwyman** [comment]
I don't see any obvious failures in your screenshot.  Perhaps it was related to the issues with repo.typesafe.com described in #449?
Regardles, as a1lu says, Djinni isn't getting any more feature work, so if you different packaging options I'd encourage you to look into providing them yourself.

**Comment by khoben** [comment]
I faced the same problem. the project was not built during the whole day. But in the end, some tricks were found that helped reduce assembly time to a few minutes. Firstly, I tried with https://github.com/LedgerHQ/djinni, the time for downloading packages on my Windows machine was reduced, but the process of resolving dependencies was still very long. The next step was to use WSL bash and disable Windows Defender, and the total build time was about ten minutes (instead of a whole day). I leave binary jar below. Good luck!
[binary .jar](https://github.com/khoben/djinni/blob/master/binary/djinni.jar)
